CMakeLists.txt

```diff
@@ -975,7 +981,7 @@ endfunction()
 
 file_glob_recurse(FILES_NEED_FORMAT DIRECTORIES      "src" "modules" "python" "test" "benchmark"
                                     PATTERNS         ".*\\.(cc|cpp|h|hpp|vineyard-mod)$"
-                                    EXCLUDE_PATTERNS ".*\\.vineyard.h$"
+                                    EXCLUDE_PATTERNS "(.*\\.vineyard.h$)|(.*modules/kv-state-cache/radix-tree/radix\.(cc|h)$)"
```

Comment:
No such rules,

- if we make no modification on it, it would be moved into the `thirdparty` directory.
- otherwise, use `// clang-format on/off` annotations inside the file

Reply:
Actually, the rule is to avoid the check of `cpplint`, and I don't find similar annotation to work around.